lib/services/event_service.ex

```diff
@@ -82,6 +82,27 @@ defmodule Aprb.Service.EventService do
                         }]",
           unfurl_links: true
          }
+      "conversations" ->
+        if event["properties"]["buyer_outcome"] == "other" do
+          %{
+            text: ":phone: #{event["object"]["display"]} responded on https://radiation.artsy.net/accounts/2/conversations/#{event["properties"]["radiation_conversation_id"]}",
```

Comment:
I would add a helper function in the future for generating these, since building up long strings with nested hash values feels a little wonky.